setup.py

```diff
@@ -19,7 +19,7 @@
 history = open("CHANGES.rst").read()
 
 tests_require = [
-    "pytest-reana>=0.8.1,<0.9.0",
+    "pytest-reana>=0.9.0a1,<0.10.0",
```

Comment:
Note: depends on two PRs and PyPI releases:
- https://github.com/reanahub/pytest-reana/pull/95
- https://github.com/reanahub/reana-commons/pull/299